.github/workflows/e2e_android.yml

```diff
@@ -0,0 +1,138 @@
+#
+# Detox e2e tests in CI
+#
+
+name: '[e2e] Android Detox Test Run'
+
+on:
+  workflow_dispatch:
+  # Run on weekdays 4:00 AM UTC
+  schedule:
+    - cron: '0 4 * * 1,2,3,4,5'
```

Comment:
Let's take this out until it actually passes. No need to ping slack or run cron just for it to fail

Reply:
@timwright12 I have taken it out. However these tests are not passing locally. I confirmed with @alexandec and @rbontrager that a decent number for iOS and Android are broken. Also unfortunately once one test breaks it breaks all the tests that occur after it.

I have moved this back to draft and ticket https://github.com/department-of-veterans-affairs/va-mobile-app/issues/6879 is created for fixing the broken e2e tests that I found

Reply:
Might want to hop on a zoom with @rbontrager to investigate your set up. Every test outside of appointments was passing locally at last check

Reply:
@timwright12 We were able to update detox and the platform error is no longer occurring. The issue now is that locally we have a different resolution/screen size than what we can enable on the emulator/simulator. So @rbontrager is now adding some scrolling to get most of the remaining tests to pass. We also came across a problem when changing json mock data in the files as well that seems to be a detox issue.